<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add object</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Object</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>Object:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="objectsComboBox"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Name:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="nameLineEdit"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Size</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <layout class="QFormLayout" name="formLayout_10">
                     <item row="0" column="0">
                      <widget class="QLabel" name="xLabel_2">
                       <property name="text">
                        <string>x:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="xSizeLineEdit"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_11">
                     <item row="0" column="0">
                      <widget class="QLabel" name="yLabel_3">
                       <property name="text">
                        <string>y:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="ySizeLineEdit"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_12">
                     <item row="0" column="0">
                      <widget class="QLabel" name="zLabel_2">
                       <property name="text">
                        <string>z:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="zSizeLineEdit"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="colorLabel">
                 <property name="text">
                  <string>Color</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <layout class="QFormLayout" name="formLayout_4">
                   <item row="0" column="0">
                    <widget class="QLabel" name="rLabel">
                     <property name="text">
                      <string>r:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="rLineEdit">
                     <property name="toolTip">
                      <string>Enter red value</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout_5">
                   <item row="0" column="0">
                    <widget class="QLabel" name="gLabel">
                     <property name="text">
                      <string>g:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="gLineEdit">
                     <property name="toolTip">
                      <string>Enter green value</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout_6">
                   <item row="0" column="0">
                    <widget class="QLabel" name="bLabel">
                     <property name="text">
                      <string>b:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="bLineEdit">
                     <property name="toolTip">
                      <string>Enter blue value</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="positionLabel">
                 <property name="text">
                  <string>Position</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <layout class="QFormLayout" name="formLayout">
                   <item row="0" column="0">
                    <widget class="QLabel" name="xLabel">
                     <property name="text">
                      <string>x:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="xLineEdit">
                     <property name="toolTip">
                      <string>Enter x value</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout_2">
                   <item row="0" column="0">
                    <widget class="QLabel" name="yLabel">
                     <property name="text">
                      <string>y:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="yLineEdit">
                     <property name="toolTip">
                      <string>Enter y value</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout_3">
                   <item row="0" column="0">
                    <widget class="QLabel" name="zLabel">
                     <property name="text">
                      <string>z:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="zLineEdit">
                     <property name="toolTip">
                      <string>Enter z value</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="rotationLabel">
                 <property name="text">
                  <string>Rotation</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <layout class="QFormLayout" name="formLayout_7">
                   <item row="0" column="0">
                    <widget class="QLabel" name="rLabel_2">
                     <property name="text">
                      <string>R:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="rollLineEdit">
                     <property name="toolTip">
                      <string>Enter roll value</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout_8">
                   <item row="0" column="0">
                    <widget class="QLabel" name="pLabel">
                     <property name="text">
                      <string>P:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="pitchLineEdit">
                     <property name="toolTip">
                      <string>Enter pitch value</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout_9">
                   <item row="0" column="0">
                    <widget class="QLabel" name="yLabel_2">
                     <property name="text">
                      <string>Y:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="yawLineEdit">
                     <property name="toolTip">
                      <string>Enter yaw value</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Surface</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Knot vector type:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="knotVectComboBox"/>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="nameSurfLineEdit"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <layout class="QFormLayout" name="formLayout_13">
                 <item row="0" column="0">
                  <widget class="QLabel" name="widthLabel">
                   <property name="text">
                    <string>w:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="widthLineEdit">
                   <property name="toolTip">
                    <string>Enter width</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayout_14">
                 <item row="0" column="0">
                  <widget class="QLabel" name="heightLabel">
                   <property name="text">
                    <string>h:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="heightLineEdit">
                   <property name="toolTip">
                    <string>Enter height</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayout_15">
                 <item row="0" column="0">
                  <widget class="QLabel" name="orderLabel">
                   <property name="text">
                    <string>Order:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="orderLineEdit">
                   <property name="toolTip">
                    <string>Enter order</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <widget class="QLabel" name="colorLabel_2">
                   <property name="text">
                    <string>Color</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <layout class="QFormLayout" name="formLayout_19">
                     <item row="0" column="0">
                      <widget class="QLabel" name="rLabel_3">
                       <property name="text">
                        <string>r:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="rLineEdit_2">
                       <property name="toolTip">
                        <string>Enter red value</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_20">
                     <item row="0" column="0">
                      <widget class="QLabel" name="gLabel_2">
                       <property name="text">
                        <string>g:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="gLineEdit_2">
                       <property name="toolTip">
                        <string>Enter green value</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_21">
                     <item row="0" column="0">
                      <widget class="QLabel" name="bLabel_2">
                       <property name="text">
                        <string>b:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="bLineEdit_2">
                       <property name="toolTip">
                        <string>Enter blue value</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_14">
                 <item>
                  <widget class="QLabel" name="positionLabel_2">
                   <property name="text">
                    <string>Position</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_12">
                   <item>
                    <layout class="QFormLayout" name="formLayout_22">
                     <item row="0" column="0">
                      <widget class="QLabel" name="xLabel_4">
                       <property name="text">
                        <string>x:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="xLineEdit_2">
                       <property name="toolTip">
                        <string>Enter x value</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_23">
                     <item row="0" column="0">
                      <widget class="QLabel" name="yLabel_5">
                       <property name="text">
                        <string>y:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="yLineEdit_2">
                       <property name="toolTip">
                        <string>Enter y value</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_24">
                     <item row="0" column="0">
                      <widget class="QLabel" name="zLabel_4">
                       <property name="text">
                        <string>z:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="zLineEdit_2">
                       <property name="toolTip">
                        <string>Enter z value</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_15">
                 <item>
                  <widget class="QLabel" name="rotationLabel_2">
                   <property name="text">
                    <string>Rotation</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_13">
                   <item>
                    <layout class="QFormLayout" name="formLayout_25">
                     <item row="0" column="0">
                      <widget class="QLabel" name="rLabel_4">
                       <property name="text">
                        <string>R:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="rollLineEdit_2">
                       <property name="toolTip">
                        <string>Enter roll value</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_26">
                     <item row="0" column="0">
                      <widget class="QLabel" name="pLabel_2">
                       <property name="text">
                        <string>P:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="pitchLineEdit_2">
                       <property name="toolTip">
                        <string>Enter pitch value</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_27">
                     <item row="0" column="0">
                      <widget class="QLabel" name="yLabel_6">
                       <property name="text">
                        <string>Y:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="yawLineEdit_2">
                       <property name="toolTip">
                        <string>Enter yaw value</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
